src/Spryker/Zed/RabbitMq/RabbitMqDependencyProvider.php

```diff
@@ -62,4 +65,18 @@ protected function addGuzzleClient(Container $container)
 
         return $container;
     }
+
+    /**
+     * @param \Spryker\Zed\Kernel\Container $container
+     *
+     * @return \Spryker\Zed\Kernel\Container
+     */
+    protected function addConnection(Container $container): Container
+    {
+        $container->set(static::CONNECTION, static function () use ($container): ConnectionInterface {
```

Comment:
Please describe why it's static

Reply:
It allows avoiding the potential problems with circular links and memory leaks.